sounddevice.py

```diff
@@ -2266,9 +2266,27 @@ def reset(self):
     # This object shadows the 'default' class, except when building the docs.
     default = default()
 
-
```

Comment:
This empty line was on purpose: [E302](http://pycodestyle.readthedocs.io/en/latest/intro.html#error-codes).